render-utils.js

```diff
@@ -17,16 +17,32 @@ export function renderDogCard(dog) {
     return a;
 }
 
-export function renderDogDetail(dog) {
+export function renderDogDetail(dogData) {
     const div = document.createElement('div');
     const img = document.createElement('img');
     const descriptionEl = document.createElement('p');
     const nameEl = document.createElement('p');
     const ageEl = document.createElement('p');
     const breedEl = document.createElement('p');
     const ageAndBreedEl = document.createElement('div');
+    img.src = `../assets/${dogData.dog_breeds.name}.jpeg`;
 
-    // examine the css classes already written and finish writing a detail page using the above elements
+     // adding classlists
+    ageAndBreedEl.classList.add('age-and-breed');
+    div.classList.add('dog-detail');
+    descriptionEl.classList.add('description');
+    nameEl.classList.add('name');
+    img.classList.add('img');
+    
 
+    // examine the css classes already written and finish writing a detail page using the above elements
+    descriptionEl.textContent = dogData.description;
+    nameEl.textContent = dogData.name;
+    ageEl.textContent = dogData.age;
+    breedEl.textContent = dogData.breed;
```

Comment:
your dogData object doesn't have a 'breed' key/value pair. the 'breed' is in the dog_breeds table under the 'name' column. you can access the breed here by targeting dogData.dog_breeds.name.